Source/Libraries/GSF.Security/AdoSecurityProvider.cs

```diff
@@ -780,11 +780,9 @@ protected virtual void LogAuthenticationAttempt(bool loginSuccess)
                 // a read-only database is being used or current user only has read-only access to database.
                 if (!string.IsNullOrWhiteSpace(SettingsCategory) && UseDatabaseLogging)
                 {
-                    AdoDataConnection database = new AdoDataConnection(SettingsCategory);
-
-                    using (IDbConnection connection = database.Connection)
+                    using (AdoDataConnection connection = new AdoDataConnection(SettingsCategory))
                     {
-                        connection.ExecuteNonQuery(database.ParameterizedQueryString("INSERT INTO AccessLog (UserName, AccessGranted) VALUES ({0}, {1})", "userName", "accessGranted"), UserData.Username, loginSuccess ? 1 : 0);
+                        connection.ExecuteNonQuery("INSERT INTO AccessLog (UserName, AccessGranted) VALUES ({0}, {1})", UserData.Username, loginSuccess ? 1 : 0);
```

Comment:
`AdoDataConnection` resolves boolean parameters under the covers.

https://github.com/GridProtectionAlliance/gsf/blob/4d453a383e4387e29ab2dfaeedd9da0bda42d1a6/Source/Libraries/GSF.Core/Data/AdoDataConnection.cs#L885-L896

```suggestion
                        connection.ExecuteNonQuery("INSERT INTO AccessLog (UserName, AccessGranted) VALUES ({0}, {1})", UserData.Username, loginSuccess);
```